### ISSUE TITLE
domSide.js

### ISSUE THREAD

**Issue by styse** [armandoalonso/c3IDE#88]
This is a great IDE for c3addons. Adding a section to create domside.js files makes it perfect. 🔥

**Comment by armandoalonso** [comment]
Yeah, unfortunately, I will not be updating c3ide any longer. There is a great project by skymen that has a much better framework for creating add-ons, and I also think he is working on an editor.

I would highly check those out...
https://github.com/ConstructFund/c3ide2-framework

There is also a cli that will bootstrap it for you and you can currently use vscode.

**Comment by styse** [comment]
Great! Thanks for the suggestion.💙

**Comment by skymen** [comment]
I would recommend checking out the cli https://github.com/ConstructFund/c3ide2-cli
It does scaffolding for you as well as a few more things.

There is an editor planned in the future but it isn't there yet. Since the whole framework is still being built, there is very little documentation about it. Feel free to either create issues on the repo and I'll do my best to help. If you prefer discord, I am active on the Construct Community Discord server where you can ask any question in the addons development channel
https://discord.com/invite/construct-community-116497549237551109

**Comment by styse** [comment]
> I would recommend checking out the cli https://github.com/ConstructFund/c3ide2-cli It does scaffolding for you as well as a few more things.
> 
> There is an editor planned in the future but it isn't there yet. Since the whole framework is still being built, there is very little documentation about it. Feel free to either create issues on the repo and I'll do my best to help. If you prefer discord, I am active on the Construct Community Discord server where you can ask any question in the addons development channel https://discord.com/invite/construct-community-116497549237551109

Nice! Thanks. This will help me a lot.🔥💙
